platform/packages/platform/src/bank_ibc/remote.rs

```diff
@@ -73,6 +73,7 @@ fn new_msg(
         receiver: receiver.into(),
         timeout_height: None,
         timeout_timestamp: timeout.nanos(),
+        memo: "".into(),
```

Comment:
Prefer `String::new()`, `String::default()` or worst-case scenario /as when reviewing in GitHub type is not clear/ `Default::default()`, as they are specifically documented to not allocate, while the `From<&str>` implementation **might not** specialize for the empty string case.